src/renderer/components/query.jsx

```diff
@@ -157,6 +157,34 @@ export default class Query extends Component {
     });
   }
 
+  getCommands () {
```

Comment:
I didn't think so as the rest of interface might not be affected by the change only the editor...

If we apply in menu, for me it might give the felling that whole app should increase the font, and it's not the case...